src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  const baseprice = days * PRICE_PER_DAY;
```

Comment:
Const can't be used here 
basePrice - new word should start with a capital letter

Reply:
agree, pay attention to your case. camelCase in the best option for javascript